models/offer.js

```diff
@@ -0,0 +1,65 @@
+const { Schema, model } = require('mongoose')
+const {
+  enums: { SPOKEN_LANG_ENUM, SUBJECT_LEVEL_ENUM }
+} = require('~/consts/validation')
+const {
+  roles: { STUDENT, TUTOR }
+} = require('~/consts/auth')
+
+const offerSchema = new Schema(
+  {
+    price: {
+      type: Number,
+      required: [true, 'This field cannot be empty.'],
+      min: [1, 'Price must be positive number']
+    },
+    proficiencyLevel: {
+      type: String,
+      enum: SUBJECT_LEVEL_ENUM,
+      required: [true, 'This field cannot be empty.']
+    },
+    description: {
+      type: String,
+      minlength: [1, 'Last Name cannot be shorter than 1 symbol.'],
+      maxlength: [200, 'Last Name cannot be longer than 300 symbols.'],
+      required: [true, 'This field cannot be empty.']
+    },
+    languages: {
+      type: [String],
+      enum: SPOKEN_LANG_ENUM,
+      required: [true, 'Spoken language must be selected.']
```

Comment:
remove spoken, its irrelevant here. write something like 
Please select a language(s) that will be used in teaching.